src/utils.js

```diff
@@ -40,6 +44,42 @@ export function validateSpacingProps(props: SpacingProps) {
   return true
 }
 
+function getMediaQuery(
+  range: string,
+  displayAliases: { [string]: string }
+): string {
+  const response = []
+  const [from, to] = displayAliases[range].split('-').map(toInt)
+
+  if (from) {
+    response.push(`(min-width: ${from}px)`)
+  }
+  if (to) {
+    response.push(`(max-width: ${to}px)`)
+  }
+  return response.join(' and ')
```

Comment:
This is one of the limitations of this approach. Right now the only media queries handled are `min/max width`)